CHANGELOG/v1.8.0-beta.1.md

```diff
@@ -0,0 +1,416 @@
+🚨 This is a BETA RELEASE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+
+## Highlights
+* Add kubeadm v1beta4 types
+* Add support negative polarity conditions
+
+<details>
+<summary>More details about the release</summary>
+
+:warning: **BETA RELEASE NOTES** :warning:
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.27.x -> v1.30.x
+- Workload Cluster: v1.25.x -> v1.30.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.7.0
```

Comment:
A few more (just merged):
* #10912
* #10849 (merging)

Reply:
PR #10912 and  #10849 added, #10922 isn't because it wasn't merged

Reply:
Added now

Reply:
@vishalanarase I don't see 10922

Reply:
Please also add 10918

Reply:
Added

Reply:
Thx!